git-remote-hg

```diff
@@ -265,10 +266,10 @@ class Parser:
         return (user, int(date), -tz)
 
 def fix_file_path(path):
-    path = os.path.normpath(path)
+    path = posixpath.normpath(path)
     if not os.path.isabs(path):
```

Comment:
I test that branch and have success in cloning.
May be, this method not use or correctly use slash against backslash. As experiment I change all "os.path" to "posixpath" but this my local commit not better fix bug than branch of razaraz:
windows. He is first author and his minimal change fix issue fully, additionan replaces "os.path" to "posixpath" don't possitive effect.
May full changes can fix other problems but it fix of other problems good do in other commits for it, as I think.